docs/index.md

```diff
@@ -84,8 +84,13 @@ Only users with access to the Admin APIs can access the Admin SDK Directory API,
 must be set in the environment variable `GOOGLEWORKSPACE_IMPERSONATED_USER_EMAIL` or in the `impersonated_user_email` attribute in the provider. Additionally, the user must have logged in at least once and accepted the Google Workspace Terms of Service.
 
 ### Using Specific Administrator Roles
+
 You do not need to set up domain-wide delegation if you are granting more specific administrator roles to the service account. If the Terraform pipeline execution environment provides an appropriate token as Application Default Credentials (ADC), you can use the provider without any further setup.
 
+To do this it's recommended that you create a custom admin role(s) with the Admin API privileges you need, as [pre-built administrator roles](https://support.google.com/a/answer/2405986) might not cover your use case.
+
+~> Some resources controlled via this provider can only be managed by a Super Admin, so user impersonation is necessary in those cases. For instance, [Admin Roles can only managed via the console or API by a Super Admin](https://support.google.com/a/answer/2406043?hl=en). Also, access to some API endpoints may not be possible to grant as privileges in custom roles. 
```

Comment:
`~>` Results in formatting like this:

<img width="706" alt="Screenshot 2022-06-08 at 19 43 40" src="https://user-images.githubusercontent.com/15078782/172692870-c732a767-d292-40a5-8aae-f0b20408da73.png">